### ISSUE TITLE
Implement Download Options for Newsstand and Opt-In

### ISSUE THREAD

**Issue by odrobnik** [Cocoanetics/AutoIngest#7]
Add download options for Newsstand and Opt-In.

I haven't found anybody with Opt-In reports yet, but it is implemented in DTITCReportDownloader as in outlined in [Apple's PDF Guide](http://www.apple.com/itunesnews/docs/AppStoreReportingInstructions.pdf)

The total options to download should be:
- Sales Summary Daily
- Sales Summary Weekly
- Sales Summary Monthly
- Sales Summary Yearly
- Newsstand Detailed Daily
- Newsstand Detailed Weekly
- Opt-In Summary Daily (?)
- Opt-In Summary Weekly (?)

Newsstand does not seem to have longer periods than Weekly at present. I assume that the same is true for Opt-In. Only Newsstand has Detailed.

I'm thinking that a matrix would be nice. There is no combination where Summary and Detailed can occur at the same time so imagine a grid, where the Os are checkboxes

```
          D    W    M    Y
Sales     O    O    O    O
Newsstand O    O    -    -
Opt-In    -    O    -    -
```

Looks like Opt-In is only available weekly.

**Comment by fcy** [comment]
I like the matrix idea

**Comment by odrobnik** [comment]
I know that there is a Mac control for Matrix I think, or would we use an NSTableView? Anybody who can implement this?

**Comment by odrobnik** [comment]
Implemented in merge 5b2ca077f27d9893756f490f60a9f00eb9918f66. 

I am hoping that somebody who has Opt-In and Newsstand reports can help test it.
